core/scheduler/service/replay_worker.go

```diff
@@ -82,18 +82,18 @@ func (w ReplayWorker) createMissingRuns(ctx context.Context, replayReq *schedule
 	for _, run := range replayReq.Runs {
 		if _, ok := existedRunsMap[run.ScheduledAt]; !ok {
 			// create any missing runs
-			if err := w.scheduler.CreateRun(ctx, replayReq.Replay.Tenant(), replayReq.Replay.JobName(), run.ScheduledAt, scheduler.StateReplayed.String()); err != nil {
+			if err := w.scheduler.CreateRun(ctx, replayReq.Replay.Tenant(), replayReq.Replay.JobName(), run.ScheduledAt, "replayed"); err != nil {
 				return nil, err
 			}
-			run.State = scheduler.StateReplayed
+			run.State = scheduler.StateInProgress
 			createdRuns = append(createdRuns, run)
 		}
 	}
 	return createdRuns, nil
 }
 
 func (w ReplayWorker) processNewReplayRequest(ctx context.Context, replayReq *scheduler.ReplayWithRun, jobCron *cron.ScheduleSpec) (err error) {
-	state := scheduler.ReplayStateReplayed
+	state := scheduler.ReplayStateInProgress
```

Comment:
you might want to verify this as this will cause all replay requests with parallel mode always in `in progress` state